random-beans/src/test/java/io/github/benas/randombeans/beans/WildCardCollectionBean.java

```diff
@@ -0,0 +1,550 @@
+/*
+ * The MIT License
+ *
+ *   Copyright (c) 2016, Mahmoud Ben Hassine (mahmoud.benhassine@icloud.com)
+ *
+ *   Permission is hereby granted, free of charge, to any ? extends Runnable obtaining a copy
```

Comment:
Hi Mahmoud,

do know how that happened, but it's very funny indeed :laughing:.

Cheers,
Pascal